internal/tshelpers/custom_commands.go

```diff
@@ -302,6 +305,50 @@ func ExpandFile(ts *testscript.TestScript, neg bool, args []string) {
 	}
 }
 
+// IsoSizeMin `isoPath` `size` checks if the specified ISO is larger
+// than the specified number of bytes.
+func IsoSizeMin(ts *testscript.TestScript, neg bool, args []string) {
+	if len(args) != 2 {
+		ts.Fatalf("usage: IsoSizeMin isoPath size")
+	}
+
+	workDir := ts.Getenv("WORK")
+	isoPath := args[0]
+	size, err := strconv.ParseInt(args[1], 10, 64)
+	ts.Check(err)
+
+	isoPathAbs := filepath.Join(workDir, isoPath)
+	fileInfo, err := os.Stat(isoPathAbs)
+	ts.Check(err)
+
+	fileSize := fileInfo.Size()
+	if fileSize < size {
+		ts.Fatalf("%s of size %d is less than %d", isoPath, fileSize, size)
+	}
+}
+
+// IsoSizeMax `isoPath` `size` checks if the specified ISO is smaller
+// than the specified number of bytes.
+func IsoSizeMax(ts *testscript.TestScript, neg bool, args []string) {
+	if len(args) != 2 {
+		ts.Fatalf("usage: IsoSizeMa isoPath size")
+	}
+
+	workDir := ts.Getenv("WORK")
```

Comment:
I think we could eliminate some duplicate code by having a IsoSize function that just returns the size of the file and then have IsoSizeMin and IsoSizeMax call it. WDYT?

Reply:
Changed to use function.